### PR TITLE
Invitees-limit function implemented + validation added on form for session-expiry

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

Changes made include:
1. Update scheduled job after editing chomp-sessions form

2. Perform job when user-limit hits

3. Added validation for chomp-sessions form creation

image

### CODE REVIEW

File: app/controllers/chomp_sessions_controller.rb [hunk 1]
```diff
@@ -71,10 +73,15 @@ def result
     redirect_to restaurant_path(@restaurant)
   end
 
+  def loading
+    sleep 10
```
Comment (TeeWhyKay) [inline app/controllers/chomp_sessions_controller.rb]: do remove this if its not used?

Comment (tjw0612) [reply app/controllers/chomp_sessions_controller.rb]: removed

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_09_28_080921) do
+
```
Comment (TeeWhyKay) [inline db/schema.rb]: sry, im ocd about this part, remove empty line 13

File: config/routes.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@
   end
 
   resources :responses, only: [:edit]
-
+
```
Comment (TeeWhyKay) [inline config/routes.rb:21]: random empty line

File: app/controllers/responses_controller.rb [hunk 4]
```diff
@@ -13,6 +13,11 @@ def create
     @response.longitude = coords[:longitude]
     if @response.save
       ResponseMailer.with(response: @response, chomp_session: @chomp_session).create_response.deliver_later if user_signed_in?
+      # check invitees limit, check if user inputted any user-limit and if responses count already exceeded
+      if @chomp_session.invitees != nil && @chomp_session.responses.count == (@chomp_session.invitees + 1)
+        GenerateRestaurantJob.perform_now(@chomp_session.id)
```
Comment (TeeWhyKay) [inline app/controllers/responses_controller.rb:18]: if there's a way to get the scheduled job to perform now, that would be even better!

**Review by TeeWhyKay — APPROVED** [review]
looks fine, remove dump.rdb

**Review by TeeWhyKay — APPROVED** [review]
looks great! Thanks

**Pull request merged**